### PR TITLE
new activity page

### PR DESCRIPTION
**Pull request by heyitsmebev**
Ticket: https://github.com/GSA/notifications-admin/issues/1677

The intention with this PR is to create a new activity page where users can view all jobs created. 

The page consist of: 
- [x] Pagination
- [x] Table with fields: **Job ID# | Template | Time sent | Sender | Report**
- [x] Rows of items within a job, page size is 50
- [x] Users should be able to **sort**
- [x] New side nav menu item: **Activity** 

Figma design: https://www.figma.com/proto/aAWhlPzOP6kfVFnCCAfSZZ/notify.gov?node-id=3047-7482&t=WNfyj0s1XDaaS0cc-0&scaling=min-zoom&content-scaling=fixed&page-id=3047%3A4396&starting-point-node-id=3047%3A5482
image

image

### CODE REVIEW

**Comment by jonathanbobel** [comment]
Was checking on the colors for the sorting active state and think we should use 
- blue-20v (#a1d3ff) for the <th> 
- blue-5v (#e8f5ff) for the <td>

Everything else looks good!

File: app/utils/pagination.py [hunk 1]
```diff
@@ -29,3 +29,15 @@ def generate_previous_next_dict(view, service_id, page, title, url_args):
         "title": title,
         "label": "page {}".format(page),
     }
+
+
+def generate_pagination_pages(total_items, page_size, current_page):
+    total_pages = (total_items + page_size - 1) // page_size
```
Comment (ccostino) [inline app/utils/pagination.py:35]: Was `// page_size` meant to be a comment?  If so, the `//` needs to be a `#` instead. 🙂 

```python
    total_pages = (total_items + page_size - 1) # page_size
```

Or were you trying to divide?  In which case, it looks like there's an extra `/` then:

```python
    total_pages = (total_items + page_size - 1) / page_size
```

**Review by ccostino — COMMENTED** [review]
Thank you, @heyitsmebev! 🎉 

Everything looks good to me, there's only one spot that I caught something specific that I wanted to check on before we merged to test in staging.

File: app/utils/pagination.py (hunk 1 above)
Comment (heyitsmebev) [reply app/utils/pagination.py:35]: https://www.geeksforgeeks.org/floor-division-in-python/

I'm using floor division, which divides the first number by the second number and then rounds down to the nearest whole number.

Comment (ccostino) [reply app/utils/pagination.py:35]: Totally missed that and forgot about it, thank you @heyitsmebev! 😀

**Review by ccostino — APPROVED** [review]

**Pull request merged**